src/layouts/shared/page-header.tsx

```diff
@@ -16,13 +20,20 @@ export default function PageHeader() {
               className={style.pageHeader__logoLink}
               data-testid="homeButton"
             >
-              <MonoIcon
-                icon="logo/logo"
-                alt=""
-                role="none"
-                size="normal"
-                overrideMode="dark"
-              />
+              {fylkeskommune && (
+                <Image
+                  width={0}
+                  height={0}
+                  sizes="100vw"
+                  style={{ width: '100%', height: 'auto' }}
```

Comment:
This is insane. But apparently best practice: https://github.com/vercel/next.js/discussions/18474#discussioncomment-5501724